### DISCUSSION TITLE
Lucy Adjustment

### DISCUSSION THREAD

**Discussion by han-tyumi** [gleam-lang/gleam#2638] [Ideas & suggestions]
First off, congrats on 1.0! 🎉
I've been looking for essentially a FP language equivalent of Go for a long while and Gleam ticks all the boxes.
The amazing community is the cherry on top.

Looking at the new website,
I just felt like Lucy needed to be slightly tilted to be more cozy with the G in Gleam. ☺️

> Before
> <img width="153" alt="image" src="https://github.com/gleam-lang/gleam/assets/13040345/2b04c884-e7d1-444b-885c-f5eee44186d9">

> After
> <img width="153" alt="image" src="https://github.com/gleam-lang/gleam/assets/13040345/8c274abe-c6e1-4699-ad55-3244e2e3f9ba">

I was also thinking a subtle floating effect (bobbing up and down) could be a nice touch.
Adding a little more personality to a great mascot.

**Comment by han-tyumi** [comment]
Well it seems it has been adjusted since I last looked. 👍

<img width="162" alt="image" src="https://github.com/gleam-lang/gleam/assets/13040345/e5abce81-4957-4ede-92dd-73df4cdb234d">

**Reply by lpil** [reply]
Sorry I forgot to reply after I took your feedback onboard. Super busy day. Thank you!

**Reply by han-tyumi** [reply]
No problem!
